tests/test_zarr_ipfs.py

```
@@ -137,3 +137,73 @@ def test_encryption(random_zarr_dataset: tuple[str, xr.Dataset]):
+
+
+    # Test with wrong API Key
+    hamt = HAMT(
+        store=IPFSStore(      
+            rpc_uri_stem = "http://127.0.0.1:5002",     
+            api_key="badKey",
+        ),
+        transformer_encode=encrypt,
+        transformer_decode=decrypt,
+    )
+    with pytest.raises(Exception):
```

Ruff (0.8.2)

`pytest.raises(Exception)` should be considered evil

(B017)